### PR TITLE
Add. XibName

### PR DESCRIPTION
**Pull request by igorkislyuk**

### CODE REVIEW

File: LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+extension UIViewController {
```
Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift]: public?

Comment (igorkislyuk) [inline LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift]: Anyway for now it's too late. But why?

Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift]: why not protocol conformance extension https://github.com/TouchInstinct/LeadKit/blob/master/LeadKit/LeadKit/Protocols/StaticNibNameProtocol.swift
like this one: https://github.com/TouchInstinct/LeadKit/blob/master/LeadKit/LeadKit/Extensions/UIView/UIView%2BDefaultNibName.swift

Comment (NikAshanin) [inline LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift]: nibName - default var for viewController. So, it does not work.

Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/UIViewController/UIViewController+XibName.swift]: hmm. ok. then I will rename protocol var in my PR

**Pull request merged**